app/assets/javascripts/draft_form_selectors.coffee

```diff
@@ -1,30 +1,53 @@
 $(document).ready ->
+  ###
```

Comment:
I am unsure that everything between here and the UMM-V stuff is all UMM-C.  Some of it may be common.  For instance, I suspect that the related URLs stuff is shared.

Reply:
I removed the label. We can reorganize and add back the label correctly another time.